build/Targets/Settings.props

```diff
@@ -118,10 +118,12 @@
        - Embed PDBs to make it easier to debug Jenkins crash dumps.
 
       Developer build:
-       - Embed PDBs to be consistent with Jenkins builds.
+       - Build Windows PDBs in order to provide stack traces in xUnit on .NET Framework (4.7.1)
+       - TODO: Use embedded PDBs once .NET Framework 4.7.2 is released and xUnit runner includes configuration switch to enable Portable PDBs
```

Comment:
Please put link to follow-up issue, instead of "TODO".

Reply:
Done.